lib/server.js

```diff
@@ -1,21 +1,29 @@
 "use strict";
 
-//Modules required
-var Hapi = require("hapi");
+var Hapi              = require("hapi");
+var Bluebird          = require("bluebird");
+var MongoMapper       = require("genesis").MongoMapper;
+var DatabaseConnector = require("./services/DatabaseConnector");
 
-//Setting the port for the application
-var port = process.env.PORT || 8080;
+var databaseConnector = DatabaseConnector.connect(process.env.DB_URL, process.env.DB_USER, process.env.DB_PASSWORD);
```

Comment:
While abstraction is appreciated, do you think this particular structure is productive? Considering the functionality of what is actually happening on this line, does encapsulating that in `DatabaseConnector.connect` make things more or less readable?